web-marketplace/src/pages/Dashboard/MyProjects/MyProjects.constants.ts

```diff
@@ -2,6 +2,7 @@ import { msg } from '@lingui/macro';
 
 export const DRAFT_ID = 'draft';
 export const CREATE_POST = msg`+ create post`;
-export const CREATE_POST_DISABLED_TOOLTIP = msg`You cannot make posts because either the project is still a draft or it doesn't have assigned a location.`;
+export const CREATE_POST_TOOLTIP_TEXT = msg`You cannot make posts because either the project is still a draft or it doesn't have assigned a location.`;
+export const EDIT_PROJECT_TOOLTIP_TEXT = msg`This is not yet supported by Wallet Connect and Keplr mobile. Please log in using Keplr browser extension on desktop.`;
```

Comment:
maybe we could rename this to NOT_SUPPORTED_TOOLTIP_TEXT or something similar since this is not just about editing a project